collections/Douce/MS_Douce_311.xml

```diff
@@ -30,61 +30,559 @@
+                        <msItem>
+                           <locus>(fol. 115v)</locus>
+                           <title type="desc">St Andrew</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 115v–116v)</locus>
+                           <title type="desc">St Christopher</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 116v–117v)</locus>
+                           <title type="desc">St Sebastian</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 117v)</locus>
+                           <note>Blank, ruled</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 118r)</locus>
+                           <title type="desc">St Mary Magdalene</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 118r)</locus>
+                           <title type="desc">St Martha</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 118v–119r)</locus>
+                           <title type="desc">St Catherine</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 119r–v)</locus>
+                           <title type="desc">St Barbara</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 119v)</locus>
+                           <title type="desc">St Margaret</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 119v–120a verso)</locus>
+                           <title type="desc">St Apollonia</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 120a verso–120d verso)</locus>
+                           <note>Blank, ruled</note>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 121r–v)</locus>
+                        <title type="desc">Short prayers</title>
+                        <textLang mainLang="la" otherLangs="frm">Rubrics in Middle French</textLang>
+                     </msItem>
+                     <msItem>
+                        <title type="desc">Prayers to Christ</title>
+                        <msItem>
+                           <locus>(fol. 122r)</locus>
+                           <note>Blank</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 122v–124r)</locus>
+                           <title type="desc">Prayer to Christ</title>
+                           <textLang mainLang="frm">Middle French</textLang>
```

Comment:
might be worth referring to Sonet,  Répertoire d'incipit des prières françaises à la fin du Moyen Âge

Reply:
It looks as if we need to get a copy of this for Weston!